config/xinitrc

```diff
@@ -0,0 +1,24 @@
+xset -dpms
+xset s off
+xset s noblank
+
+# need to figure out brightness
+
+unclutter -idle 0.01 -root &
+
+if [ "${VX_MACHINE_TYPE}" = bmd ]; then
+    URL=http://localhost:3000/speech-loader.html
+else
+    URL=http://localhost:3000/
+fi
+
+exec sh <<SHELL
```

Comment:
This works? I’d have expected `exec sh <<SHELL` for it to be terminated by `SHELL` later.

Reply:
I think something got mangled here. I'll check :)

Reply:
no, actually, not mangled, yes this works. I'm not sure I understand what you think it might do?

Reply:
Oh. Weird. This was an issue with the Github iOS app. It only showed `exec sh <`.